packages/discord.js/src/managers/RoleManager.js

```diff
@@ -169,20 +178,22 @@ class RoleManager extends CachedManager {
    * Edits a role of the guild.
    * @param {RoleResolvable} role The role to edit
    * @param {RoleData} data The new data for the role
-   * @param {string} [reason] Reason for editing this role
    * @returns {Promise<Role>}
    * @example
    * // Edit a role
-   * guild.roles.edit('222079219327434752', { name: 'buddies' })
-   *   .then(updated => console.log(`Edited role name to ${updated.name}`))
+   * guild.roles.edit('222079219327434752', {
```

Comment:
imo this change isn't needed as reason isn't a role data so it shouldn't belong to `roleData` or u need to change the typedef name to `roleEditData` but I won't suggest it. Also you need to edit the typedef also
https://github.com/discordjs/discord.js/blob/861f0e2134662ab64a11d313130aff58b413d495/packages/discord.js/src/structures/Role.js#L193